client/flutter/lib/pages/license_page.dart

```diff
@@ -552,80 +93,44 @@ class _LicensePageState extends State<LicensePage> {
       _loaded = true;
     });
     assert(() {
-      Timeline.timeSync('Build scheduled', () { }, flow: Flow.end(debugFlowId));
+      Timeline.timeSync('Build scheduled', () {}, flow: Flow.end(debugFlowId));
       return true;
     }());
   }
 
   @override
   Widget build(BuildContext context) {
-    assert(debugCheckHasMaterialLocalizations(context));
-    final String name = widget.applicationName ?? _defaultApplicationName(context);
-    final String version = widget.applicationVersion ?? _defaultApplicationVersion(context);
-    final Widget icon = widget.applicationIcon ?? _defaultApplicationIcon(context);
-    final MaterialLocalizations localizations = MaterialLocalizations.of(context);
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(localizations.licensesPageTitle),
-      ),
-      // All of the licenses page text is English. We don't want localized text
-      // or text direction.
-      body: Localizations.override(
-        locale: const Locale('en', 'US'),
-        context: context,
-        child: DefaultTextStyle(
-          style: Theme.of(context).textTheme.caption,
-          child: SafeArea(
-            bottom: false,
-            child: Scrollbar(
-              child: ListView(
-                padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 12.0),
-                children: <Widget>[
-                  Text(name, style: Theme.of(context).textTheme.headline5, textAlign: TextAlign.center),
-                  if (icon != null) IconTheme(data: Theme.of(context).iconTheme, child: icon),
-                  Text(version, style: Theme.of(context).textTheme.bodyText2, textAlign: TextAlign.center),
+    return Localizations.override(
+      locale: const Locale('en', 'US'),
```

Comment:
What's this about?

Reply:
This was in the original, I assumed it was because the licenses were only available in english.